test/cypress/features/verifyMessage.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Verify Message
+
+  Scenario: Verify the integrity of a signed message in textarea
+    Given I login as genesis on devnet
+    And I wait 2 seconds
+    And I open signMessage modal
+    When I fill test_Message in signMessageInput field
+    And I click on nextBtn
+    And I click on copyToClipboardBtn
+    And I open verifyMessage modal
+    And I click on textAreaViewBtn
+    Then I verify signed message in verifyMessageTextArea
+
+  Scenario: Verify the integrity of a tampered message in textArea view
+    Given I login as genesis on devnet
+    And I wait 2 seconds
+    And I open verifyMessage modal
+    And I click on textAreaViewBtn
+    When I paste invalid_text in verifyMessageTextArea field
```

Comment:
instead of `I paste`, I think I `I fill` should surface meaning the I paste test definition in the definition file can be removed